### DISCUSSION TITLE
Should Get-DbaHelpIndex output FillFactor property?

### DISCUSSION THREAD

**Discussion by Jonathan-Quilter** [dataplat/dbatools#9192] [Contributors]
Hi, just gone to dbatools to collect fill_factor info and cant see the property output by Get-DbaHelpIndex. 

Looking in the repo the property value is pulled from the [database object](https://github.com/dataplat/dbatools/blob/b4ce4b332aac0af0a33080ea4f5089381daacb10/public/Get-DbaHelpIndex.ps1#L981), its just not included in either of the final output objects [here](https://github.com/dataplat/dbatools/blob/b4ce4b332aac0af0a33080ea4f5089381daacb10/public/Get-DbaHelpIndex.ps1#L1045) or [here](https://github.com/dataplat/dbatools/blob/b4ce4b332aac0af0a33080ea4f5089381daacb10/public/Get-DbaHelpIndex.ps1#L1081). 

Should it be there? Has it been excluded for a reason?

**Comment by wsmelton** [comment]
It does look like it should be output and can't see in history where it was added or previously removed.

**Reply by wsmelton** [reply]
Created issue from this for fixing the command (linked on the right under the `Events` section of GitHub UI).
